modules/labs-react/text-input/lib/TextInput.Field.tsx

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+
+import {borderRadius, inputColors, spaceNumbers, type} from '@workday/canvas-kit-react/tokens';
+import {
+  createComponent,
+  errorRing,
+  ErrorType,
+  ExtractProps,
+  styled,
+  StyledType,
+  Themeable,
+  useModelContext,
+} from '@workday/canvas-kit-react/common';
+import {Box} from '@workday/canvas-kit-labs-react/common';
+
+import {TextInputModelContext} from './TextInput';
+import {TextInputModel} from './useTextInputModel';
+
+export interface TextInputFieldProps extends ExtractProps<typeof Box, 'input'>, Themeable {
```

Comment:
Other components do not include the element interface since it is added again in use. Change the `'input'` to `never` to not automatically add them. What happens is TS will add all the element attributes in this interface, then remove them all only to add them all again.

```suggestion
export interface TextInputFieldProps extends ExtractProps<typeof Box, never>, Themeable {
```

`ExtractProps` was made to add them again if someone wants to extend our components with a baked interface. For example:

```ts
type SomeProps = ExtractProps<typeof TextInput.Field> // TextInputFieldProps & React.InputHTMLAttributes<HTMLInputElement>
```

The `React.InputHTMLAttributes<HTMLInputElement>` attributes are added by `ExtractProps` based on the default element passed into `createComponent`.

Reply:
Ah thanks forgot about that bit, will update